website/content/en/preview/getting-started-with-terraform/_index.md

```diff
@@ -106,15 +106,10 @@ module "eks" {
       asg_max_size  = 1
     }
   ]
-
-  tags = {
-    "karpenter.sh/discovery" = var.cluster_name
-  }
```

Comment:
We need this one, too

Reply:
:roll_eyes: ouch, seems that came in from rebasing earlier, fixing now!